src/app_engine/constants.py

```diff
@@ -15,7 +16,13 @@
 TURN_URL_TEMPLATE = '%s/turn?username=%s&key=%s'
 CEOD_KEY = '4080218913'
 
-WSS_HOST_PORT_PAIR = 'apprtc-ws.webrtc.org:443'
+WSS_HOST_PORT_PAIRS = ['apprtc-ws.webrtc.org:443', 'apprtc-ws-2.webrtc.org:443']
+WSS_HOST_STATUS_LIST_KEY = 'wss_host_status_list'
+
+WSS_HOST_INSTANCE_KEY = 'instance'
+WSS_HOST_STATUS_KEY = 'status'
+WSS_HOST_STATUS_CODE_KEY = 'status_code'
```

Comment:
OK, I see it now. Issue solved.

On Wed, Feb 25, 2015 at 2:28 PM, Chuck Hays <notifications@github.com>
wrote:

> In src/app_engine/constants.py
> <https://github.com/webrtc/apprtc/pull/92#discussion_r25386200>:
>
> > @@ -15,7 +16,13 @@
> >  TURN_URL_TEMPLATE = '%s/turn?username=%s&key=%s'
> >  CEOD_KEY = '4080218913'
> >
> > -WSS_HOST_PORT_PAIR = 'apprtc-ws.webrtc.org:443'
> > +WSS_HOST_PORT_PAIRS = ['apprtc-ws.webrtc.org:443', 'apprtc-ws-2.webrtc.org:443']
> > +WSS_HOST_STATUS_LIST_KEY = 'wss_host_status_list'
> > +
> > +WSS_HOST_INSTANCE_KEY = 'instance'
> > +WSS_HOST_STATUS_KEY = 'status'
> > +WSS_HOST_STATUS_CODE_KEY = 'status_code'
>
> They are still written out on line 167. handle_collider_response creates a
> result object for each instance, which are added to an array. The array is
> written out on 167.
>
> Do you think we should do something different? I was trying to keep a
> human readable output so that you can call /probe/collider from a browser
> and see the current status.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/apprtc/pull/92/files#r25386200>.
>



-- 
Jiayang Liu